mix.exs

```diff
@@ -2,7 +2,7 @@ defmodule Tokenizers.MixProject do
   use Mix.Project
 
   @source_url "https://github.com/elixir-nx/tokenizers"
-  @version "0.3.2"
+  @version "0.13.0"
```

Comment:
I am not so sure if we should tie ourselves to the tokenizers version. The others are all officially maintained by Hugging Face, so it is understandable, but we should probably give us a bit more space. :)

```suggestion
  @version "0.4.0-dev"
```